lib/rouge/lexers/jsl.rb

```diff
@@ -14,24 +14,24 @@ class JSL < RegexLexer
         rule %r/\s+/m, Text::Whitespace
 
         rule %r(//.*?$), Comment::Single
-        rule %r'/[*].*', Comment::Multiline, :comment
+        rule %r'/[*].*[*]/', Comment::Multiline # single line block comment
+        rule %r'/[*].*', Comment::Multiline, :comment # multiline block comment
```

Comment:
We can solve this with the existing rule by making the `.*` not greedy: 

```ruby
rule %r'/[*].*', Comment::Multiline, :comment
```

Reply:
Please also add an example of this single line comment syntax to the visual sample.